.github/ISSUE_TEMPLATE/stabilization.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Module stabilization
+about: Stabilize a module before a 1.0 release
+title: 'Stabilize module X'
+labels: 'stabilization'
+assignees: ''
+---
+
+Before stabilizing a module, an approver or maintainer must make sure that the following criteria are met:
+
+- [ ] One RC release or more have been done of this module
```

Comment:
Do we want to do rc releases going forward, given the reported issues?

Reply:
I think doing one or two RC releases makes sense, but I agree that definitely 18 RC releases is too much :smile: We can add some guidance to this document as part of solving #8063 (something like "if we do more than two RCs go back to minor versions until we solve the issue(s) in question")

Reply:
Maybe we can just require two 0.x versions with all other checkboxes marked instead of jumping back and forth to RC

Reply:
unless there's a strong argument for a RC tag, i would agree that we could call a package out as RC as of version 0.x and avoid unnecessary process and pain